ainft/chat/messages.py

```diff
@@ -0,0 +1,125 @@
+from __future__ import annotations
+
+import json
+from typing import List
+
+from ain.ain import Ain
+
+from ..types import (
+    SetOperation,
+    SetMultiOperation,
+    TransactionInput,
+    Message,
+    MessageTransactionResult,
+)
+from ..utils import *
+
+
+class Messages:
+    _max_content_length = 200
+
+    def __init__(self, ain: Ain):
+        self._ain = ain
+
+    async def store(
+        self,
+        *,
+        messages: List[Message],
+        object_id: str,
+        token_id: str,
+    ) -> MessageTransactionResult:
+        """
+        Store a list of messages.
+
+        Args:
+            messages: The list of messages.
+
+            object_id: The ID of the AINFT object.
+
+            token_id: The ID of the AINFT token.
+        """
+        app_id = get_app_id(object_id)
+        user_addr = validate_user_address(self._ain.wallet)
+
+        await self._validate(app_id, token_id, user_addr, messages)
+
+        return await self._send_tx_for_store_message(
+            **dict(
+                messages=messages,
+                app_id=app_id,
+                token_id=token_id,
+                address=user_addr,
+            )
+        )
+
+    async def _validate(
+        self, app_id: str, token_id: str, address: str, messages: List[Message]
+    ):
+        await validate_app(app_id, self._ain.db)
+        await validate_token(app_id, token_id, self._ain.db)
+        for message in messages:
+            await validate_thread(
+                app_id, token_id, address, message.thread_id, self._ain.db
+            )
+            validate_message_id(message.id)
+
+    async def _send_tx_for_store_message(self, **kwargs) -> MessageTransactionResult:
+        timestamp = int(now())
+        tx_body = self._build_tx_body_for_store_message(timestamp=timestamp, **kwargs)
+        tx_result = await self._ain.sendTransaction(tx_body)
+
+        if not is_tx_success(tx_result):
+            raise RuntimeError(f"Failed to send transaction: {json.dumps(tx_result)}")
+
+        return self._format_tx_result(tx_result, **kwargs)
+
+    def _build_tx_body_for_store_message(
+        self,
+        app_id: str,
+        token_id: str,
+        messages: List[Message],
+        address: str,
+        timestamp: int,
+    ) -> TransactionInput:
+        op_list = []
+        for msg in messages:
+            msg_key = str(msg.created_at)
```

Comment:
@jakepyo timestamp is used as the message key (e.g. `messages/{msg_key}`), 
so please input each message's creation time in [`create_at`](https://github.com/ainft-team/ainft-py/pull/4/files#diff-69d1708b62236393c02cf7188d2b9a0b18c61832d887fe2f35ce2a429307a7d0R53).